swatch-contracts/src/test/java/com/redhat/swatch/contract/service/export/SubscriptionDataExporterServiceTest.java

```diff
@@ -273,6 +281,79 @@ void testErrorWhenServiceReturnsTimeout() {
     thenErrorLogWithMessage("Error handling export request");
   }
 
+  @Test
+  public void testJsonMapDataItem() {
```

Comment:
These tests are not covering all the export logic (query and mapping), but only the mapping.
Instead of adding new tests, you could simply populate the billing account ID you expect in the method `givenSubscriptionWithMeasurement`, and assert that the billing account ID is populated.